src/main/java/frc/robot/subsystems/DriveBase.java

```diff
@@ -51,10 +51,16 @@ private DriveBase() {
         rightFrontTalon = new WPI_TalonSRX(Config.RIGHT_FRONT_TALON);
         rightRearTalon = new WPI_TalonSRX(Config.RIGHT_REAR_TALON);
 
+        talon5plyboy = new WPI_TalonSRX((Config.TALON_5_PLYBOY));
+
         robotDriveBase = new DifferentialDrive(leftFrontTalon, rightFrontTalon);
-        _pidgey = new PigeonIMU (Config.robotSpecific(null, null, null, leftFrontTalon, leftRearTalon));
 
-        _pidgey.setFusedHeading(0.0, 30);
+
+        var pigeonTalon = Config.robotSpecific(null, null, rightRearTalon, leftFrontTalon, leftRearTalon, talon5plyboy);
+        if(pigeonTalon != null){
+            _pidgey = new PigeonIMU (pigeonTalon);
```

Comment:
do we need to check _pidgey is non-null as well before calling setFusedHeading?

Reply:
From what we have tested, its not needed to detect that for the setFusedHeading